test/failed-directories.test.js

```diff
@@ -83,7 +89,71 @@ test('do not find file, and give up', function (assert) {
   });
 });
 
-test('stop at stopDir, and give up', function (assert) {
+test('do not find file, tries at user\'s home, and give up', function (assert) {
+  setup();
+  var startDir = absolutePath('a/b');
+  readFileStub = sinon.stub(fs, 'readFile', function (searchPath, encoding, callback) {
+    switch (searchPath) {
+      case absolutePath('a/b/package.json'):
+      case absolutePath('a/b/.foorc'):
+      case absolutePath('a/b/foo.config.js'):
+      case absolutePath('a/package.json'):
+      case absolutePath('a/.foorc'):
+      case absolutePath('a/foo.config.js'):
+      case absolutePath('package.json'):
+      case absolutePath('.foorc'):
+      case absolutePath('foo.config.js'):
+      case atHomeDir('package.json'):
+      case atHomeDir('.foorc'):
+      case atHomeDir('foo.config.js'):
+        callback({ code: 'ENOENT' });
+        break;
+      default:
+        callback(new Error('irrelevant path ' + searchPath));
+    }
+  });
+
+  var loadConfig = cosmiconfig('foo', {
+    stopDir: absolutePath('.'),
```

Comment:
If this is the config, I don't understand why we're looking in the home directory at all. The searching should stop in this very `test/` directory, right?